homeassistant/components/influxdb.py

```diff
@@ -120,7 +120,11 @@ def influx_event_listener(event):
 
         for key, value in state.attributes.items():
             if key != 'unit_of_measurement':
-                json_body[0]['fields'][key] = value
+                if isinstance(value, (str, float, bool)):
+                    json_body[0]['fields'][key] = value
+                elif isinstance(value, int):
```

Comment:
Why was this needed? Integers are allowed and handled by Influxdb, AFAIK. I think we should have left this for another PR as it's a breaking change.